server/app.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+const app = express();
+const cors = require('cors')
+
+app.use(cors());
+app.use(express.json());
+
+const search = require('./data')
+
+
+app.get('/', (req, res) => {
+    res.json('It works!')
+})
+
+//Reading aspects of the code
+
+app.get('/search', (req, res) => {
+    res.json(search)
+})
+
+app.get('/search/:name', (req, res) => {
+    try {
+        let requestedSearch = req.params.name
+        let matchingSearch = search.find((item) => item.title.toLowerCase() === requestedSearch.toLowerCase());
+        if(!matchingSearch) { throw new Error(`Sorry we don't have a page called ${requestedSearch}`)}
+        res.json(matchingSearch)
+    } catch (error) {
+        res.status(404).json({message: error.message});
```

Comment:
For me, this worked on the server-side but not on client-side:

![Screenshot 2022-01-27 at 09 47 36](https://user-images.githubusercontent.com/88836794/151334162-70c908dd-de6b-4224-a4a3-284c061a56c3.png)

Reply:
parrot wasn't an option. Guess we needed some error handling there